src/templates/day_labor/form-data.html

```diff
@@ -52,6 +52,8 @@
         <div class='form-data-section_item'>
```

Comment:
Kind of unclear if reccomendedRate would be its own section. Could a user supply an hourly rate and ALSO ask for a suggestion? I'll check w/ @desigonz